packages/note/index.ts

```diff
@@ -1,3 +1,37 @@
+type NoteLetter = "A" | "B" | "C" | "D" | "E" | "F" | "G";
+type NoteAccidental = string; // " " | "#" | "b" | " #" | " b"  | "b " | "b#" | "#b" ;
+type Note = string;
+type Midi = number;
+type Octave = number;
+type OrNull<T> = T | null;
+type NoteName = string;
+
+type NoteProps = {
+  name: NoteName;
+  letter: NoteLetter;
+  acc: NoteAccidental | string; // {String}: the note accidentals
```

Comment:
Probably this is no longer required, isn't it?
```suggestion
  acc: NoteAccidental; // {String}: the note accidentals
```